srcpkgs/radicale/template

```diff
@@ -18,8 +18,7 @@ distfiles="https://github.com/Kozea/Radicale/archive/refs/tags/v${version}.tar.g
 checksum=4247b18004f0779e00392a3fced1d93f720f50b7be30b46d0283f09de6d1e45b
 conf_files="
  /etc/radicale/config
- /etc/radicale/rights
- /etc/sv/radicale/log/run"
```

Comment:
should this line stay or get removed?

Reply:
not sure why it exists, but probably removed